bridge-worker/bridge_worker.go

```diff
@@ -0,0 +1,115 @@
+package bridgeworker
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"log"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	ethconfig "github.com/keep-network/keep-common/pkg/chain/ethereum"
+	ethconnect "github.com/mezo-org/mezod/ethereum"
+	"github.com/mezo-org/mezod/ethereum/bindings/portal"
+)
+
+// mezoBridgeName is the name of the MezoBridge contract.
+const mezoBridgeName = "MezoBridge"
+
+type BridgeWorker struct {
+	bridgeContract *portal.MezoBridge
+	chain          *ethconnect.BaseChain
+}
+
+func RunBridgeWorker(
+	providerURL string,
+	ethereumNetwork string,
+	privateKey *ecdsa.PrivateKey,
+) {
+	network := ethconnect.NetworkFromString(ethereumNetwork)
+	mezoBridgeAddress := portal.MezoBridgeAddress(network)
+
+	if mezoBridgeAddress == "" {
+		panic(
+			"cannot get address of the MezoBridge contract on Ethereum; " +
+				"make sure you run 'make bindings' before building the binary",
+		)
+	}
+
+	log.Printf(
+		"resolved MezoBridge address: %s and Ethereum network: %s",
+		mezoBridgeAddress,
+		network,
+	)
```

Comment:
Is the usage of the log package expected here? Elsewhere we use the cosmosdk.io/log one.

Reply:
oh don't get me wrong, I'm not very opinionated either way, just wanted to make sure this was a conscious choice of yours.